jena-arq/src/test/java/org/apache/jena/sparql/core/AbstractTestDynamicDataset.java

```diff
@@ -204,7 +204,17 @@ public void dynamic_dft_7() {
                   "{ GRAPH <urn:x-arq:DefaultGraph> { ?s <uri:p> ?o . FILTER ( ?o IN ( 1, 2) ) } }",
                   2, dataset);
     }
-    
+
+    @Test
```

Comment:
This class is not an ideal fit for the join tests. Any suggestion?

Reply:
Can you not make the `testCount()` method `public static` and then create a new test class that simply calls it?

Reply:
With my question I was mainly referring to whether I have overlooked an already existing suitable place for these tests - if there is none so far I could of course create a new class and register it with the test suite.

Reply:
There is now a AbstractTestQueryExec hierarchy analogous to AbstractTestDynamicDataset.